app/controllers/application_controller.rb

```diff
@@ -37,8 +37,12 @@ def routing_error
       when Common::Exceptions::BaseError
         exception
       when Common::Client::Errors::BackendServiceError
-        additional_attributes = { source: exception.developer_message, meta: exception.as_json }
-        Common::Exceptions::ClientError.new(exception.message, additional_attributes)
+        additional_attributes = {
+          detail: exception.message,
+          source: exception.developer_message,
+          meta: exception.as_json
+        }
+        Common::Exceptions::ClientError.new(nil, additional_attributes)
```

Comment:
I guess I'm still a little bit confused here, but:

* Shouldn't this error also be renamed to indicate a problem with the backend service?
* Why do we have to make this translation here? Couldn't we just raise this exception type in the first place?
* I'm sure there's a semantic difference between the classes under `Common::Client::Errors` and `Common::Exceptions`, but I'm not sure the naming makes it clear what that difference is.